pints/plot/_surface.py

```diff
@@ -0,0 +1,324 @@
+#
+# Plots a function defined on a two-dimensional space.
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+# This module was was adapted from Myokit (see http://myokit.org) by the
+# original author.
+#
+# The code to plot voronoi regions was based on an example shown here:
+# http://stackoverflow.com/a/20678647/423420
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+import numpy as np
+
+import pints
+
+
+def surface(points, values, boundaries=None, markers='+', figsize=None):
+    """
+    Takes irregularly spaced points and function evaluations in a
+    two-dimensional parameter space and creates a coloured surface plot using a
+    voronoi diagram.
+
+    Returns a ``matplotlib`` figure object and axes handle.
+
+    Parameters
+    ----------
+    points
+        A list of (two-dimensional) points in parameter space.
+    values
+        The values corresponding to these points.
```

Comment:
'values' can be a misleading word in Pints, aren't we used to call the data to be fitted to as 'values'.

Reply:
What else would you call them? They are the values that go with some points. Could be errors or likelihoods. Don't have to :man_shrugging: 

I'm quite happy for pints methods to have an argument `value` of `values` where the value isn't a measured data point.

Reply:
I've updated the docstring to make this more clear

Reply:
Much better! Thanks! Yeah, I guess just explaining it in the doc makes it clear.